.env.sample

```diff
@@ -0,0 +1,3 @@
+ENABLE_CZ=true
+ENABLE_NPM_LINT=true
+ENABLE_HELM_DOCS=true
```

Comment:
look, enforcement of style is not optional, and we should all have the same workflow. I am against these flags as it makes devs bypass what needs to be uniformly applied.

If you have suggestions to make that better, great. But don't attempt to undermine that with such flags please.